src/lib/paginator/paginator-constants.ts

```diff
@@ -77,3 +77,13 @@ export interface IPaginatorRangeState {
 }
 
 export type PaginatorRangeLabelBuilder = ((state: IPaginatorRangeState) => string) | undefined | null;
+
+export interface IPaginatorPageInfo {
+  pageIndex: number;
+  pageSize: number;
+  total: number;
+  offset: number;
+  pageStart: number;
+  pageEnd: number;
+  totalPages: number;
+}
```

Comment:
Do we still need this?

Reply:
Nope! Good catch, I was going to expose the state as a single method but chose to revert that since we have individual properties for everything already. Updated, thanks.